### PR TITLE
[Communication]-Sms-Added optional DeliveryReportTimeoutInSeconds to smsSendOptions

### PR DESCRIPTION
**Pull request by besh2014**
Added optional deliveryReportTimeoutInSeconds to smsSendOptions. RestApi spec is [here ](https://github.com/Azure/azure-rest-api-specs/commit/8df53db16935b96733fda9882c3f197e2f961287)

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/communication/Azure.Communication.Sms/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.1.0-beta.1 (2022-02-05)
```
[the remark this answers is not in the document]
Comment (besh2014) [reply sdk/communication/Azure.Communication.Sms/CHANGELOG.md]: right, updated

File: sdk/communication/Azure.Communication.Sms/README.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ To create a new Communication Service, you can use the [Azure Portal][communicat
 ### Using statements
 ```C# Snippet:Azure_Communication_Sms_Tests_UsingStatements
 using System;
+using System.IO;
```
Comment (phermanov-msft) [inline sdk/communication/Azure.Communication.Sms/README.md:26]: Curious why System.IO is required now?

Comment (besh2014) [reply sdk/communication/Azure.Communication.Sms/README.md]: Good question, looks some automatic addition. Removed.

**Review by gfeitosa-msft — APPROVED** [review]

**Review by ilyapaliakou-msft — APPROVED** [review]

File: sdk/communication/Azure.Communication.Sms/api/Azure.Communication.Sms.netstandard2.0.cs [hunk 3]
```diff
@@ -14,15 +14,17 @@ public partial class SmsClient
...
         public enum ServiceVersion
         {
             V2021_03_07 = 1,
+            V2024_02_05_Preview = 2,
         }
     }
     public partial class SmsSendOptions
     {
         public SmsSendOptions(bool enableDeliveryReport) { }
+        public int? DeliveryReportTimeoutInSeconds { get { throw null; } set { } }
```
Comment (DominikMe) [inline sdk/communication/Azure.Communication.Sms/api/Azure.Communication.Sms.netstandard2.0.cs:27]: We're trying to use language built-in times for the SDK and then translate internally to the type for the REST API, consider using `TimeSpan` and rounding to seconds. You can also search through the repo to see how other Azure service SDKs handle time spans and if you find others who use numbers with a suffixed name instead, but I think the Azure .NET architects would prefer TimeSpan.

**Pull request merged**